### ISSUE TITLE
Foxglove Studio takes too long to start

### ISSUE THREAD

**Issue by swarajpeppermint** [foxglove/studio#7260]
**Description**
Since a few releases ago, Foxglove Studio takes too long to start up (white screen to home screen). Attaching a screen recording of the delay. During the screen recording, I just have the browser running on the system.

- Version: 1.82.0
- Platform: Ubuntu 22.04.3
- System Specs: RAM 16.0 GB, Processor 11th Gen Intel® Core™ i7-11800H @ 2.30GHz × 16
- Data source (e.g. bag file, mcap file, rosbridge, ROS 1/2 native): N/A

**Steps To Reproduce**
Start Foxglove Studio on an updated Ubuntu 22.04 system

**Screen Recording**
[Screencast from 12-19-2023 09:43:18 AM.webm](https://github.com/foxglove/studio/assets/85288063/8c04d904-3bac-47a7-b386-035a4e3cf69a)

**Expected Behavior**
Should show home screen within a second or two

**Comment by jtbandes** [comment]
Can you press ctrl+shift+i when the window appears and see if there is any information in the developer console or network tab?

**Comment by swarajpeppermint** [comment]
![image](https://github.com/foxglove/studio/assets/85288063/268f92f7-f6ee-4575-afce-c83ef8874603)

**Comment by swarajpeppermint** [comment]
Foxglove Studio started hanging after a while
![image](https://github.com/foxglove/studio/assets/85288063/7ad50279-0391-41e2-a6a8-176759217b3b)

**Comment by swarajpeppermint** [comment]
network tab
![image](https://github.com/foxglove/studio/assets/85288063/0f61ce85-a636-4927-abac-407c10370318)

**Comment by playerdefault** [comment]
@wimagguc Yes I have a custom extension installed. It's this extension: https://github.com/playerdefault/foxglove-pointstamped-viz

**Comment by playerdefault** [comment]
A good update on this. Launching foxglove studio via `foxglove-studio --disable-gpu --disable-software-rasterizer` launches it **instantly**. The same issue is happening with VSCode too. 
Relevant GitHub issue with VSCode: https://github.com/microsoft/vscode/issues/192493#issuecomment-1937098984

Maybe it's not an issue with FoxGlove but something to do with a recent update with Electron?

**Comment by defunctzombie** [comment]
Folks experiencing this issue - did you install via Snap or Deb?

**Comment by playerdefault** [comment]
> Folks experiencing this issue - did you install via Snap or Deb?

snap

**Comment by defunctzombie** [comment]
@playerdefault if you are still experiencing this with the snap install - could you give the Deb install a try and see if it exhibits the same behavior on your machine. That would be a helpful data point for us.

**Comment by playerdefault** [comment]
> @playerdefault if you are still experiencing this with the snap install - could you give the Deb install a try and see if it exhibits the same behavior on your machine. That would be a helpful data point for us.

issue is not occurring with the deb version
